### DISCUSSION TITLE
Teacher - Unable to join GitHub Education with my school's email

### DISCUSSION THREAD

**Discussion by rbelfortcsi** [community/community#120090] [GitHub Education]
### Hi everyone,

Hello! 

I have been a GitHub user while I was a student for years. Now that I am currently a technology teacher at a High School, I am happy to re-join GitHub and have my students learn about this platform.

However, when I try to join the GitHub Education program as a teacher, my school is not listed. The school I work at is **Colegio San Ignacio de Loyola** which is located in San Juan, Puerto Rico (U.S. Territory). 

My email ends with **@sanignacio.pr**.  Any suggestions as for how could I apply for GitHub Education, even if my school is not listed?

Any help would be greatly appreciated.  Thanks!

Reynaldo Belfort

**Comment by samus-aran** [comment]
Hi @rbelfortcsi  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍

**Reply by rbelfortcsi** [reply]
Thanks! Will try raising a ticket. Will let know how it goes.

All the best!

**Reply by rbelfortcsi** [reply]
So far I haven't heard anything... Any help will be greatly appretiated!

Thanks,
Reynaldo
